### PR TITLE
Remove some usages of ActorMaterializer

### PR DESCRIPTION
**Pull request by ignasi35**
There were a bunch of deprecation warning a bit annoying.

### CODE REVIEW

File: interop-tests/src/main/java/akka/grpc/interop/AkkaGrpcClientJava.java [hunk 1]
```diff
@@ -5,9 +5,6 @@
 package akka.grpc.interop;
 
 import akka.actor.ActorSystem;
-import akka.stream.ActorMaterializer;
-import akka.stream.Materializer;
-import akka.grpc.interop.GrpcClient;
 import io.grpc.internal.testing.TestUtils;
```
Comment (ignasi35) [inline interop-tests/src/main/java/akka/grpc/interop/AkkaGrpcClientJava.java:8]: These imports were not even used.

**Comment by raboof** [comment]
Change looks good, but failure seems legitimate (https://travis-ci.com/github/akka/akka-grpc/jobs/434054415#L580).

I thought we had fatal warnings enabled, but that does not seem to be taking effect for some reason.

**Comment by ignasi35** [comment]
> I thought we had fatal warnings enabled, but that does not seem to be taking effect for some reason.

Not for java:

https://github.com/akka/akka-grpc/blob/be4c8da080302864911ac2a1cb60fb4ba36fa8da/project/Common.scala#L51

I'm guessing we disabled because of these deprecation warning in generated Java code:

```
[warn] .../src_managed/main/io/grpc/reflection/v1alpha/ErrorResponse.java:88:1: io.grpc.reflection.v1alpha.ServerReflectionProto in io.grpc.reflection.v1alpha has been deprecated
```

**Comment by raboof** [comment]
> deprecation warning in generated Java code

Ah, and we have no silencer for Java. I guess this is unavoidable then.

**Comment by raboof** [comment]
```
[warn] /home/travis/build/akka/akka-grpc/build.sbt isn't formatted properly!
```

**Review by raboof — APPROVED** [review]

**Pull request merged**